doc/source/changelog.rst

```diff
@@ -3,6 +3,39 @@ Changelog
 *********
 
 
+Version 0.4.0 (Nov 23, 2024)
++++++++++++++++++++++++++++++
+
+This release adds a new quantum circuit renderer based on matplotlib and text and improves the circuit simulation speed.
+
+New features
+------------
+- Add matplotlib-based and text-based quantum circuit renderer. The new renderer is more flexible and can be used in different environments without latex dependencies. (`#239 <https://github.com/qutip/qutip-qip/pull/239>`_ and `#244 <https://github.com/qutip/qutip-qip/pull/244>`_ by Rushiraj Gadhvi)
+
+Improvements
+------------
+- `einsum` is used to speed up the circuit simulation. (`#225 <https://github.com/qutip/qutip-qip/pull/225>`_ by Boxi Li)
```

Comment:
Do you have a sense of how much the speed up was? Might be nice to say "speed up the circuit simulation by up to 2x" or similar to give the reader a sense of the improvement. Also happy to leave as is if it's too difficult to give a number.